armi/nuclearDataIO/isotxs.py

```diff
@@ -175,7 +175,7 @@ def _write(cls, lib, fileName, fileMode):
 
     @classmethod
     def _readWrite(cls, lib, fileName, fileMode, getNuclideFunc):
-        with _IsotxsIO(fileName, lib, fileMode, getNuclideFunc) as rw:
+        with cls(fileName, lib, fileMode, getNuclideFunc) as rw:
```

Comment:
Still seems kind of squirrely

Reply:
🐿️  
was squirrelier before when it was demolishing any inheritance by having the class be hard-coded!